elastic-search-support/src/main/resources/application.yml

```diff
@@ -51,6 +51,36 @@ elasticsearch:
                             }
                         }
                     }'
```

Comment:
Kiran this new mappings should be tested 1) import a new file check if mappings are there, by doing a get indexes in elastic 2) create new cases and che logstash logs and make sure that the data population was done in elastic 3)  do a elastic query that search by the values of the new fields